File: c#/task3/14mod3.cs
using System;

class Program{
    static bool IsPrime(int n){
        if (n < 2) return false;
        for (int i = 2; i * i <= n; i++){
            if (n % i == 0) return false;
        }

        return true;
    }

    // Главная функция — точка входа в программу, как main() в C++
    static void Main(){
        Console.WriteLine("Введите количество чисел:");
        int n = int.Parse(Console.ReadLine()); // Чтение и преобразование ввода из строки в int
        int count = 0;

        for (int i = 0; i < n; i++){
            Console.Write($"{i + 1} Число: ");
            int num = int.Parse(Console.ReadLine()); // Считываем каждое число

            if (IsPrime(num)){
                count++;
            }
        }

        Console.WriteLine($"{count} Простых чисел в массиве");
    }
}

File: rust/task3/14mod3.rs
use std::io;

fn is_prime(n: i32) -> bool {
    if n < 2 {
        return false;
    }
    for i in 2..=(n as f64).sqrt() as i32 {
        if n % i == 0 {
            return false;
        }
    }
    true
}

fn main() {
    let mut input = String::new();
    println!("Введите количество чисел:");
    io::stdin().read_line(&mut input).expect("Ошибка");
    let n: i32 = input.trim().parse().expect("Не число");

    let mut count = 0;
    for i in 0..n {
        input.clear();
        println!("{} Число:", i + 1);
        io::stdin().read_line(&mut input).expect("Ошибка");
        let num: i32 = input.trim().parse().expect("Не число");

        if is_prime(num) {
            count += 1;
        }
    }
    println!("{} Простых чисел", count);
}
